lib/daru.rb

```diff
@@ -95,6 +95,7 @@ def plotting_library= lib
 require 'forwardable'
 require 'erb'
 require 'date'
+require 'mechanize'
```

Comment:
According to our discussion on issue #219 , wasn't it decided that `Mechanize` should be a development dependency & not be installed along with `daru` on `gem install` right? I'm yet to catch the `LoadError` if `Mechanize` is not installed - I'll do this in the subsequent commit(s).

Specific comment : https://github.com/SciRuby/daru/issues/219#issuecomment-278578308